src/controllers/room.controller.js

```diff
@@ -0,0 +1,137 @@
+const bcrypt = require('bcryptjs')
+const parametros = require('../lib/configParameters')
+const mongoose =  require("mongoose");
+const { v4: uuidv4 } = require("uuid");
+
+const mongodbConnection = require('../databases/mongodb/repository/mongodbManager');
+const mongodbRoom = require('../databases/mongodb/models/rooms.model')
+
+// conectar a la bbdd
+var mongoDB = "mongodb://127.0.0.1/studybuddies";
+mongoose.connect(mongoDB, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useFindAndModify: false
+  });
+
+const create = async function(req, res) {
+    
+    let nErrores = 0;
+    let statusCode = 0;
+    let statusMessage = '';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7172443)
```suggestion
    let statusMessage = "";
```